tensorflow/lite/micro/kernels/xtensa/softmax_int8_int16.cc

```diff
@@ -108,11 +108,11 @@ void* XtensaInitSoftmax(TfLiteContext* context, const char* buffer,
 TfLiteStatus XtensaPrepareSoftmax(TfLiteContext* context, TfLiteNode* node) {
 #if defined(HIFI4) || defined(HIFI4_INTERNAL) || defined(HIFI5)
   return PrepareHifi(context, node);
-#elif defined(VISION_P6)
-  TF_LITE_ENSURE_OK(context, SoftmaxPrepareVision(context, node));
-  return kTfLiteOk;
 #else
```

Comment:
I would suggest creating some consistency between the prepare functions. PrepareHifi calls SoftmaxPrepare, but SoftmaxPrepareVision does not. Consider having either both call SoftmaxPrepare or neither.

Reply:
For Vision P6 we have always followed this method of calling regular prepare followed by prepareVision for all ported kernels.  will need to check with HiFi team